appveyor.yml

```diff
@@ -7,9 +7,10 @@ environment:
 install:
   - ps: Install-Product node $env:nodejs_version
   - set CI=true
-  - npm -g install npm@latest
+  - set AVA_APPVEYOR=true
+  - npm -g install npm@latest || (timeout 30 && npm -g install npm@latest)
   - set PATH=%APPDATA%\npm;%PATH%
-  - npm install
+  - npm install || (timeout 30 && npm install)
```

Comment:
> Probably because it's annoying to always have to wait for AppVeyor before merging a PR. 

Yes. That.

I got a lot of TV watched this weekend. Push a commit, watch 20 minutes... Still broken... Push a commit, watch 20 minutes...